src/utils/asset.js

```diff
@@ -131,12 +131,14 @@ export const getTransactionExplorerLink = (hash, asset, network) => {
   const chain = cryptoassets[asset].chain
   const link = `${EXPLORERS[chain][network].tx}`
 
-  return link.includes('{hash}') ? link.replace('{hash}', transactionHash) : link + transactionHash
+  return link.includes('{template}') ? link.replace('{template}', transactionHash) : link + transactionHash
```

Comment:
This is not what i meant 😅 

I mean that it's unnecessary to have 2 ways of adding the hash/address to the URL
The placeholder approach works for everything
so we can add the placeholder `{hash}` to every url in `EXPLORERS`
and then simply do `link.replace('{hash}', transactionHash)`

Reply:
applied